typescript/dynamic-workflow-executor/README.md

```diff
@@ -65,14 +58,7 @@ Run workflow service:
 npm run workflow-service
 ```
 
-Register the services at the Restate Server, using the CLI:
-
-```shell
-npx @restatedev/restate dp reg localhost:9080
-npx @restatedev/restate dp reg localhost:9081
-npx @restatedev/restate dp reg localhost:9082
-npx @restatedev/restate dp reg localhost:9083
-```
+Now [launch the Restate Server](../../README.md#launching-the-restate-server) and [register the services](../../README.md#register-the-deployment-in-restate).
```

Comment:
How do people know that for this example there are multiple deployments to be discovered? Should we keep this information here?